src/pages/rekey-reason/cat-be/__tests__/rekey-reason.cat-be.page.spec.ts

```diff
@@ -0,0 +1,247 @@
+import { CAT_B } from '../../../page-names.constants';
+import { AppInfoModel } from '../../../../modules/app-info/app-info.model';
+import {
+  IpadIssueSelected,
+  OtherSelected,
+  OtherReasonUpdated,
+  TransferSelected,
+  IpadIssueLostSelected,
+} from '../../../../modules/tests/rekey-reason/rekey-reason.actions';
+import { By } from '@angular/platform-browser';
+import { SetExaminerConducted } from '../../../../modules/tests/examiner-conducted/examiner-conducted.actions';
+import { MockComponent } from 'ng-mocks';
+import { FindUserProvider } from '../../../../providers/find-user/find-user';
+import { FindUserProviderMock } from '../../../../providers/find-user/__mocks__/find-user.mock';
+import { IpadIssueComponent } from '../../components/ipad-issue/ipad-issue';
+import { TransferComponent } from '../../components/transfer/transfer';
+import { OtherReasonComponent } from '../../components/other-reason/other-reason';
+import { NavigationStateProvider } from '../../../../providers/navigation-state/navigation-state';
+import { NavigationStateProviderMock } from '../../../../providers/navigation-state/__mocks__/navigation-state.mock';
+
+describe('RekeyReasonCatBEPage', () => {
+  let fixture: ComponentFixture<RekeyReasonCatBEPage>;
+  let component: RekeyReasonCatBEPage;
+  let loadingController: LoadingController;
+  let navContoller: NavController;
+  let modalController: ModalController;
+  let store$: Store<AppInfoModel>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        RekeyReasonCatBEPage,
+        MockComponent(IpadIssueComponent),
+        MockComponent(TransferComponent),
+        MockComponent(OtherReasonComponent),
+      ],
+      imports: [
+        IonicModule,
+        StoreModule.forRoot({
+          journal: () => ({
+            isLoading: false,
+            lastRefreshed: null,
+            slots: {},
+            selectedDate: '',
+            examiner: {
+              staffNumber: '1234567',
+            },
+          }),
+          tests: () => ({
+            currentTest: {
+              slotId: '123',
+            },
+            testStatus: {},
+            startedTests: {
+              123: {
+                vehicleDetails: {},
+                accompaniment: {},
+                testData: {
+                  dangerousFaults: {},
+                  drivingFaults: {},
+                  manoeuvres: {},
+                  seriousFaults: {},
+                  testRequirements: {},
+                  ETA: {},
+                  eco: {},
+                  vehicleChecks: {
+                    showMeQuestion: {
+                      code: 'S3',
+                      description: '',
+                      outcome: '',
+                    },
+                    tellMeQuestion: {
+                      code: '',
+                      description: '',
+                      outcome: '',
+                    },
```

Comment:
This is not a valid CAT BE test.  Was this to just create a CAT BE version that is really just a copy of CAT B or is this meant to work as a CAT BE rekey?

Reply:
This was just a copy of CAT B, will change it to be a cat BE test instead, good spot